Tests/test_polymer_conformation.py

```diff
@@ -0,0 +1,207 @@
+#  Copyright (c) 2020, Build-A-Cell. All rights reserved.
```

Comment:
I guess the license files should be 2021 now :D probably a mass find-and-replace needed.

Reply:
Lets do this on all of BioCRNpyler in the dev-->master PR that is no doubt coming soon.